public/dist/data/helper-scripts/restaurants.js

```diff
@@ -0,0 +1,33 @@
+var $restaurants = $('.restaurant-list').find('li');
+var restaurantList = [];
+
+function getrestaurantData(index) {
```

Comment:
Global function - please namespace this.

Reply:
done